### PR TITLE
samples: updated comments on recognize.js to reflect time limitations…

### PR DESCRIPTION
**Pull request by b-loved-dreamer**
Please allow this change to go though. I updated the comment on the recognize.js file to reflect time limitations on local files for the longrunningrecognize

### CODE REVIEW

File: samples/recognize.js [hunk 1]
```diff
@@ -197,6 +197,9 @@ async function asyncRecognize(
     sampleRateHertz: sampleRateHertz,
     languageCode: languageCode,
   };
+
+  // Note that transcription is limited to 60 seconds audio.
```
Comment (JustinBeckwith) [inline samples/recognize.js:201]: 👋  looks like you need to run `npm run fix` here

Comment (b-loved-dreamer) [reply samples/recognize.js]: It's fixed now.

Comment (b-loved-dreamer) [reply samples/recognize.js]: @googlebot I signed it!

**Review by JustinBeckwith — APPROVED** [review]

**Comment by bcoe** [comment]
@b-loved-dreamer thank you.

**Pull request merged**